ocw/tests/test_metrics.py

```diff
@@ -208,41 +229,6 @@ def test_function_run(self):
         expected_result.fill(-300)
         np.testing.assert_array_equal(self.mean_bias.run(self.target_dataset,self.reference_dataset), expected_result)
 
-    def test_function_run_abs(self):
-        '''Test mean bias function between reference dataset and target dataset with abs as True.'''
-        expected_result = np.zeros((5, 5), dtype=np.int)
-        expected_result.fill(300)
-        np.testing.assert_array_equal(self.mean_bias.run(self.reference_dataset, self.target_dataset, True), expected_result)
-
-
-class TestSpatialMeanOfTemporalMeanBias(unittest.TestCase):
```

Comment:
Hey @huikyole, so if we don't need the test then we don't need the metric that it's testing I gather? This sounds like something that should be under another ticket so it can be discussed outside of the scope of this ticket which seems to me to be changing the call structure of metrics so they use isolated functions that take arrays instead of OCW dataset objects. Thoughts?